src/commands/lambda/__tests__/functions/commons.test.ts

```diff
@@ -629,7 +629,9 @@ describe('commons', () => {
       ]
       const cloudWatch = makeMockCloudWatchLogs({})
 
-      await updateLambdaFunctionConfigs(lambda as any, cloudWatch as any, configs)
+      for (const config of configs) {
+        await updateLambdaFunctionConfig(lambda as any, cloudWatch as any, config)
```

Comment:
if we expect a number of configs, we should use `await Promise.all` or `await Promise.allSettled`